tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs

```diff
@@ -15,6 +15,7 @@
 using StackExchange.Redis;
 using Xunit;
 using Xunit.Abstractions;
+using Newtonsoft.Json.Linq;
```

Comment:
Can you change this to use System.Text.Json? We shouldn't be adding new code using Newtonsoft.Json.